policyengine/outputs/macro/single/gov/local_areas/parliamentary_constituencies.py

```diff
@@ -5,24 +5,25 @@
 import pandas as pd
 from typing import Callable
 from policyengine_core import Microsimulation
-from policyengine.utils.constituency_maps import plot_hex_map
-
-DEFAULT_VARIABLES = [
-    "household_net_income",
-]
+from policyengine.utils.maps import plot_hex_map
 
 
 def parliamentary_constituencies(
     simulation: Simulation,
     metric: Callable[[Microsimulation], MicroSeries] = None,
     chart: bool = False,
+    code_index: bool = False,
 ) -> dict:
     """Calculate the impact of the reform on parliamentary constituencies.
 
     Args:
         simulation (Simulation): The simulation for which the impact is to be calculated.
-        custom_function (Callable[[Microsimulation], [float]]): A custom function to calculate the impact. This must be called on a Microsimulation and return a float (we will call it for each constituency weight set).
+        metric (Callable[[Microsimulation], [float]]): A custom function to calculate the impact. This must be called on a Microsimulation and return a float (we will call it for each constituency weight set).
```

Comment:
```suggestion
        metric (Callable[[Microsimulation], MicroSeries]): A custom function to calculate the impact. This must be called on a Microsimulation and return a float (we will call it for each constituency weight set).
```

Reply:
The second is the output right? In which case I think this would be incorrect.